db/migrate/20170702021929_create_law_offices.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLawOffices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :law_offices do |t|
+      t.string :law_office
+      t.string :contact_information
```

Comment:
At the moment, it looks like you're using this column to store both phone number and address.  I think it would be nicer to instead have multiple columns for this information.  That would also make searching easier when we implement it.

Specifically, what do you think about removing the `contact_information` column, and adding columns for the following instead?
*  `phone_number`
*   `street_address`
*   `city`
*  `state`
*  `zip`